HIP/hip-904.md

```diff
@@ -1170,21 +1171,50 @@ HTS system contracts will need to support the new airdrop related transactions t
+| `0xebd595e0`           | `function rejectTokens(address rejectingAaddress, address[] memory ftAddresses,  NftID[] memory nftIDs) external returns (int64 responseCode)` | TokenReject      | `ResponseCode` | The response code from the call |
+
+New system contract functions to be included for a new `IHRC904` interface.
+
+| Function Selector Hash |                                               Function Signature                                                | HAPI Transaction | Responsible service |    Response    |                                 |
+|------------------------|-----------------------------------------------------------------------------------------------------------------|------------------|---------------------|----------------|---------------------------------|
+| `0xcef5b705`           | `function cancelAirdropFT(address receiverAddress) external returns (int64 responseCode)`                       | TokenCancel      | HTS                 | `ResponseCode` | The response code from the call |
+| `0xad4917cf`           | `function cancelAirdropNFT(address receiverAddress, int64 serialNumber) external returns (int64 responseCode)`  | TokenCancel      | HTS                 | `ResponseCode` | The response code from the call |
+| `0xa83bc5b2`           | `function claimAirdropFT(address senderAddress) external returns (int64 responseCode)`                          | TokenClaim       | HTS                 | `ResponseCode` | The response code from the call |
+| `0x63ada5d7`           | `function claimAirdropNFT(address senderAddress, int64 serialNumber) external returns (int64 responseCode)`     | TokenClaim       | HTS                 | `ResponseCode` | The response code from the call |
+| `0x76c6b391`           | `function rejectTokenFT() external returns (int64 responseCode)`                                                | TokenReject      | HTS                 | `ResponseCode` | The response code from the call |
+| `0xa869c78a`           | `function rejectTokenNFTs(int64[] memory serialNumbers) external returns (int64 responseCode)`                  | TokenReject      | HTS                 | `ResponseCode` | The response code from the call |
+| `0xf5677e99`           | `function setUnlimitedAutomaticAssociations(bool enableAutoAssociations) external returns (int64 responseCode)` | CryptoUpdate     | HAS                 | `ResponseCode` | The response code from the call |
 
 To increase API reuse and for consistency the input objects will closely match the HAPI protobuf objects.
+We will introduce the following new structures to support the new functionality:
+
+`PendingAirdrop` - A struct that represents a pending airdrop request.
+
+```solidity
+struct PendingAirdrop {
+    address sender;
+    address receiver;
+
+    address token;
+    int64 serial;
```

Comment:
It seems totally obvious that `serial` is only for NFTs - but still, maybe a comment that addresses that (and states explicitly that that depends on the token type addressed by the `token` field)?  (Reason is: It's a little odd that the very next lines introduce `NftID` for identifying "the Nft serial to be rejected" but there's nothing similar for fungible types (the address only?) and anyway it's not used here in `PendingAirdrop` because that's the way solidity works: No union types.). And must `serial == 0` for fungible tokens or is it a don't care?

Reply:
Added a comment for clarification.
For the latter comment, the stated functions are redirect calls that already have the token address so it makes it obsolete to use `NftID` there. The `address` parameters there are the token airdrop sender/receiver as stated in the naming.

Reply:
ah! my mistake